data/blog/2024/1-what.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 1) What
```

Comment:
```suggestion
title: What  to hack at ETHBerlin04
```

Reply:
I think this was intentional :) https://x.com/SBF_FTX/status/1591989554881658880